build/npm/clarity-icons-README.md

```diff
@@ -7,19 +7,13 @@
 
 2. Install the polyfill for Custom Elements:
     ```
-    npm install @webcomponents/custom-elements@1.0.0-rc.3
+    npm install @webcomponents/custom-elements@1.0.0
     ```
 
-3. If your application supports IE10, the polyfill will require the MutationObserver shim to work. If your application doesn't support IE10, you can skip the following installation:
-    ```
-    npm install mutationobserver-shim@0.3.2
-    ```
-
-4. Include the clarity-icons.min.css and clarity-icons.min.js in your HTML. As custom-elements.min.js is dependent on the Custom Elements polyfill, make sure to include it before clarity-icons.min.js. Also, if your app needs to support IE10, include the mutationobserver.min.js before the polyfill:
+3. Include the clarity-icons.min.css and clarity-icons.min.js in your HTML. As custom-elements.min.js is dependent on the Custom Elements polyfill, make sure to include it before clarity-icons.min.js:
```

Comment:
>  As custom-elements.min.js is dependent on the Custom Elements polyfill,

You have custom-elements.min.js is dependent on  the Custom Elements polyfill.
It should be Clarity Icons is dependent on the Custom Elements polyfill, right?